docs/user/dashboard/lens.asciidoc

```diff
@@ -220,3 +227,49 @@ Visualizing saved searches in unsupported in *Lens*.
 ===== How do I change the number of suggestions?
 
 Configuring the *Suggestions* that *Lens* automatically populates is unsupported.
+
+[float]
+[[is-it-possible-to-use-different-indexpatterns-in-lens]]
+===== Is it possible to use different index patterns in a single visualization?
+
+You can create a visualization from multiple index patterns in *Lens*: this feature is available for a subset of visualizations such as *Bar*, *Line* and *Area* charts.
+
+Each *Layer* in a visualization is associated with an index pattern and mutiple *Layers* can be combined together within the same visualization. Each *Layer* also has a chart switcher button in order to select the best type of visualization for the specific dataset.
+At the same time it is possible to dynamically change the indexpattern for the *Layer* when required.
+
+[float]
+[[why-my-field-x-is-missing-from-the-fields-list]]
+===== Why is my field X missing from the fields list in Lens?
+
+*Lens* does not support the visualization of full-text fields, therefore it is not showing them in the data summary.
```

Comment:
```suggestion
Creating visualizations with full-text fields is unsupported in *Lens*. When your index pattern contains a full-text field, *Lens* displays the `X.keyword` field, but is unable to display the data summary.
```

Reply:
I think the suggested version is not correct, as it assumes always the availability of the `X.keyword` field for each full-text field, while this is not true.

Reply:
How about:

Creating visualizations with full-text fields is unsupported in *Lens*. When your index pattern contains full-text fields, the fields do not appear in the data summaries.

Reply:
The bits of information to communicate, I think, are the following three:
* *Lens* does not support full-text fields
* If a full-text field has also `keyword` subfield then the latter is shown
* Data summaries are not available for full-text fields, even when the `keyword` subfield is available.

In particular the first 2 are the most important part, while the third could be omitted if we want - it's just nice as it adds an extra bit to the essential information.